### DISCUSSION TITLE
Installing (Old Style with Python 2)

### DISCUSSION THREAD

**Discussion by dawson6** [llnl/ATS#6] [General]
Shawn documenting how he is currently installing ATS with Python 2.

This is the Python 2 method. See the README file.

**Comment by dawson6** [comment]
Discussion Points with Chris and David on the above

1. The GitHub install is minimal, but provides a working ATS which can schedule and run jobs using Slurm and LSF. 
2. LSF has options for both lrun and jsrun
3. Demonstrate HelloATS
4. Demonstrate HelloGPU
5. Demonstrate use of atslite1 with MC and input decks

GitHub is the place for work on any scheduler integration (including Flux) both on the LC and any other machine.  It is appopriate for Open Source work.  While it h as enough LC machine knowledge to run jobs, it does not have other LLNL specific extensions

GitLab has an ATSLLNL project which extensions to the ATS core. 

Both repos have a single setup-py2 install script, meant to be run to install in the same location.

**Comment by white238** [comment]
Just a quick note because I said I would try this today. I forgot you needed a custom installed, user-writable python. Then I ran out of time.  I will pick this back up soon.
